### ISSUE TITLE
Question about calculating depth loss.

### ISSUE THREAD

**Issue by sweemicLSY** [OPEN-AIR-SUN/mars#52]
Hello, your work is great.
but it seems that you mutiply depth_gt with directions_norm twice. Does it make sense?
the default value of `self.config.is_euclidean_depth` is `False` at Line 112, nsg/models/scene_graph.py
```
    is_euclidean_depth: bool = False
    """Whether input depth maps are Euclidean distances (or z-distances)."""
```
at Line 695-696, nsg/models/scene_graph.py, you mutiply depth_gt with directions_norm the first time.
```
            if not self.config.is_euclidean_depth:
                depth_gt = depth_gt * outputs["directions_norm"]
```
at Line 715-720, nsg/models/scene_graph.py, in fuction `monosdf_depth_loss`, you mutiply depth_gt with directions_norm the second time.
```
            mono_depth_loss = monosdf_depth_loss(
                termination_depth=depth_gt,
                predicted_depth=predicted_depth,
                is_euclidean=self.config.is_euclidean_depth,
                directions_norm=outputs["directions_norm"],
            )
```
the fuction in nsg/model_components/loss.py
```
def monosdf_depth_loss(
    termination_depth: Float[Tensor, "*batch 1"],
    predicted_depth: Float[Tensor, "*batch 1"],
    directions_norm: Float[Tensor, "*batch 1"],
    is_euclidean: bool,
):
    """MonoSDF depth loss"""
    if not is_euclidean:
        termination_depth = termination_depth * directions_norm
    sift_depth_loss = ScaleAndShiftInvariantLoss(alpha=0.5, scales=1)
    mask = torch.ones_like(termination_depth).reshape(1, 32, -1).bool()
    return sift_depth_loss(predicted_depth.reshape(1, 32, -1), (termination_depth * 50 + 0.5).reshape(1, 32, -1), mask)
```
Is it propal?

**Comment by xBeho1der** [comment]
Hi! Thank you very much for your attention to our work.

Please refer to our [paper and supplementary](https://arxiv.org/pdf/2307.15058.pdf) for more information about depth supervision.

![image](https://github.com/OPEN-AIR-SUN/mars/assets/107318439/21555272-3e3c-478f-b059-23e8c3285a3a)

```python
            loss_dict["depth_loss"] = (
                self.config.depth_loss_mult * depth_loss + self.config.mono_depth_loss_mult * mono_depth_loss
            )
```

As you can see, you should specify the `self.config.depth_loss_mult` and `self.config.mono_depth_loss_mult` in `cicai_config.py` for different depth data. You can see a example config of using mono depth in #38 .
